Jenkinsfile

```diff
@@ -23,6 +23,11 @@ node() {
   docker.image('blueocean_build_env').inside("--net=container:blueo-selenium") {
     withEnv(['GIT_COMMITTER_EMAIL=me@hatescake.com','GIT_COMMITTER_NAME=Hates','GIT_AUTHOR_NAME=Cake','GIT_AUTHOR_EMAIL=hates@cake.com']) {
       try {
+        stage('Sanity check dependencies') {
```

Comment:
I do not remember why we had the `Sanity check dependencies` in the end but AFAIR it had been because you had to do the `npm i` before, but I may be wrong.

Reply:
it was because if you had version mismatches, things went wrong. i just moved it because it's fast and annoying to wait for an entire build/test to run before finding out this fails

Reply:
yeah it never worked until towards the end, which was always annoying